docker-id/index.md

```diff
@@ -0,0 +1,45 @@
+---
+description:  Docker ID
+keywords:
+- accounts, docker ID, billing, paid plans, support, Cloud, Hub, Store, Forums, knowledge base, beta access
+title: Docker ID Accounts
+---
+
+Your free Docker ID grants you access to Docker services such as the Docker
+Store, Docker Cloud, Docker Hub repositories, and some beta programs. Your Docker ID becomes repository namespace used by hosted services such as Docker Hub and Docker Cloud. All you need is an email address.
+
+This account also allows you to log in to services such as the Docker Support
+Center, the Docker Forums, and the Docker Success portal.
+
+
+## Register for a Docker ID
+
+Your Docker ID becomes your user namespace for hosted Docker services, and becomes your username on the Docker Forums. A Docker ID also
+
+1. Go to the [Docker Cloud sign up page](https://cloud.docker.com).
+
+2. Enter a username, a valid email address, and a password.
+
+3. Click **Sign Up**.
+
+  Docker sends a verification email to the address you provided.
+
+  > **Note**: You must click this link to activate your account. You cannot log in with your Docker ID until you verify your email address.
```

Comment:
Seems funny to say you must click the link before telling them to click the link in the next step?